### PR TITLE
Removed html inline Styles and removed destroyed auto formatting

### PR DESCRIPTION
**Pull request by nborschke**
Signed-off-by: nborschke <nicoborschke98@gmail.com>
### Description

Removed html inline Styles from all js files that are directly under the js folder.
Also removed destroyed auto formatting

Fixes #

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Review by shucon — CHANGES_REQUESTED** [review]

File: js/error_report.js [hunk 1]
```diff
@@ -122,8 +122,7 @@ var ErrorReport = {
         ErrorReport._removeErrorNotification();
 
         var $div = $(
-            '<div style="position:fixed;bottom:0;left:0;right:0;margin:0;' +
-            'z-index:1000" class="error" id="error_notification"></div>'
+            '<div class="error" id="error_notification"></div>'
```
Comment (mauriciofauth) [inline js/error_report.js]: I think you should use `class` here instead of `id`.

File: js/functions.js [hunk 2]
```diff
@@ -3560,13 +3560,13 @@ AJAX.registerOnload('functions.js', function () {
         }
         var seeMore = '';
         if (list_size > maxRows) {
-            seeMore = '<fieldset class=\'tblFooters center\' style=\'font-weight:bold\'>' +
+            seeMore = '<fieldset id=\'seeMoreFieldSet\' class=\'tblFooters center\'>' +
```
Comment (mauriciofauth) [inline js/functions.js]: I think you should use `class` here instead of `id`.

File: js/functions.js [hunk 3]
```diff
@@ -3560,13 +3560,13 @@ AJAX.registerOnload('functions.js', function () {
...
                 '<a href=\'#\' id=\'seeMore\'>' + PMA_messages.seeMore + '</a></fieldset>';
         }
-        var central_columns_dialog = '<div style=\'max-height:400px\'>' +
+        var central_columns_dialog = '<div id=\'central_columns_dialog\'>' +
```
Comment (mauriciofauth) [inline js/functions.js]: I think you should use `class` here instead of `id`.

File: js/server_status_monitor.js [hunk 4]
```diff
@@ -1156,13 +1156,12 @@ AJAX.registerOnload('server_status_monitor.js', function () {
         var tempTooltipContentEditor = function (str, seriesIndex, pointIndex, plot) {
             var j;
             // TODO: move style to theme CSS
-            var tooltipHtml = '<div style="font-size:12px;background-color:#FFFFFF;' +
-                'opacity:0.95;filter:alpha(opacity=95);padding:5px;">';
+            var tooltipHtml = '<div id="tooltipHtml">';
```
Comment (mauriciofauth) [inline js/server_status_monitor.js]: I think you should use `class` here instead of `id`.

File: js/server_status_monitor.js [hunk 5]
```diff
@@ -1156,13 +1156,12 @@ AJAX.registerOnload('server_status_monitor.js', function () {
...
             // x value i.e. time
             var timeValue = str.split(',')[0];
             var seriesValue;
             tooltipHtml += 'Time: ' + timeValue;
-            tooltipHtml += '<span style="font-weight:bold;">';
+            tooltipHtml += '<span id="span1">';
```
Comment (mauriciofauth) [inline js/server_status_monitor.js]: I think you should use `class` here instead of `id`.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
Hi, @nborschke. Thanks for your contribution.
You should revert the changes generated by the automatic formatter.
You should also add your changes to the other themes.

**Pull request closed without merging**